src/Bridge/Doctrine/Persister/ObjectManagerPersister.php

```diff
@@ -27,35 +27,45 @@ class ObjectManagerPersister implements PersisterInterface
 {
     use IsAServiceTrait;
 
-    private $objectManager;
+    /** @var ObjectManager[]  */
+    private $managers;
 
     /**
-     * @var array|null Values are FQCN of persistable objects
+     * @var ClassMetadata[] Entity metadata, FQCN being the key
      */
-    private $persistableClasses;
+    private $metadata = [];
 
     /**
-     * @var ClassMetadata[] Entity metadata, FQCN being the key
+     * @var string[ObjectManager] Keys are FQCN of persistable objects, Values are managers
      */
-    private $metadata = [];
+    private $classManagerMap;
 
-    public function __construct(ObjectManager $manager)
+    public function __construct(ObjectManager ...$managers)
```

Comment:
technically, this allows to pass no manager at all (a variable argument is 0+, not 1+)

Reply:
Well, if you want to keep a variadic constructor, you will need an argument to add all the relevant references as arguments in your service definition.